### PR TITLE
Add ready/disconnect event typings

### PR DESCRIPTION
**Pull request by DusterTheFirst**
ready and disconnect events from the client were missing

### CODE REVIEW

**Comment by aurieh** [comment]
You also have to implement the ready event in Client and Shard classes, as `Emittable` is an interface.

**Comment by DusterTheFirst** [comment]
bump?

**Comment by abalabahaha** [comment]
Trying to find a Typescript user to review :|

File: index.d.ts [hunk 1]
```diff
@@ -317,7 +318,8 @@ declare module "eris" {
     connected_accounts: Array<{ verified: boolean, type: string, id: string, name: string }>;
   }
 
-  interface Connection {
```
Comment (aurieh) [inline index.d.ts]: Was this intentional?

**Review by aurieh — CHANGES_REQUESTED** [review]

File: index.d.ts (hunk 1 above)
Comment (DusterTheFirst) [reply index.d.ts]: no

**Pull request merged**